src/utils/transaction.ts

```diff
@@ -934,6 +1137,210 @@ export const buildCrossChainAction = async (
     }
   }
 
+  if (
+    transactionBlock.type === TRANSACTION_BLOCK_TYPE.HONEY_SWAP_LP &&
+    !!transactionBlock?.values?.fromChainId &&
+    !!transactionBlock?.values?.toToken1 &&
+    !!transactionBlock?.values?.toToken2 &&
+    !!transactionBlock?.values?.fromAssetAddress &&
+    !!transactionBlock?.values?.fromAssetDecimals &&
+    !!transactionBlock?.values?.fromAssetSymbol &&
+    !!transactionBlock?.values?.amount &&
+    !!transactionBlock?.values?.routeToUSDC &&
+    !!transactionBlock?.values?.offer1 &&
+    !!transactionBlock?.values?.offer2 &&
+    !!transactionBlock?.values?.tokenOneAmount &&
+    !!transactionBlock?.values?.tokenTwoAmount
+  ) {
+    try {
+      const {
+        values: {
+          fromChainId,
+          fromAssetAddress,
+          fromAssetDecimals,
+          fromAssetSymbol,
+          fromAssetIconUrl,
+          amount,
+          accountType,
+          routeToUSDC,
+          receiverAddress,
+          offer1,
+          offer2,
+          tokenOneAmount,
+          tokenTwoAmount,
+          toToken1,
+          toToken2,
+        },
+      } = transactionBlock;
+
+      const USDC_GNOSIS = '0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83';
```

Comment:
Yeah there's like 9 instances we're using this same address - better to have as an export imo